app/views/items/_form.html.erb

```diff
@@ -52,6 +52,12 @@
               <%= f.input :visible, label: "Item is Visible to Partners?", wrapper: :input_group do %>
                 <%= f.check_box :visible_to_partners, {class: "input-group-text", id: "visible_to_partners"}, "true", "false" %>
               <% end %>
+
+              <% if current_user.has_role?(Role::ORG_ADMIN, current_organization) || current_user.has_role?(Role::ORG_USER, current_organization) %>
```

Comment:
I'm pretty sure this check is not necessary because only org users can get to this page. Same with the other partials. @cielf can you confirm?

Reply:
Only bank users can see this page, so we don't need a finer-grained control at this time.

Reply:
Noted. I will remove those check.